api/app/tests/apps/sigmax/test_outgoing.py

```diff
@@ -56,3 +56,15 @@ def test_generate_creeerZaak_Lk01_priority_high(self):
             '<ZKN:einddatumGepland>{}</ZKN:einddatumGepland>'.format(
                 incident_date_end.astimezone(current_tz).strftime('%Y%m%d')),
             xml_message)
+
+
+class TestGenerateOmschrijving(TestCase):
+    def setUp(self):
+        self.signal = SignalFactoryValidLocation()
+        self.signal.priority.priority = Priority.PRIORITY_HIGH
```

Comment:
You can do this directly in the factory creation as well. WIth the current implementation the priority is not saved to the database, only saved on the `self.signal` instance.

`self.signal = SignalFactoryValidLocation(priority__priority=Priority.PRIORITY_HIGH)`